package.json

```diff
@@ -25,7 +25,7 @@
     "lint:dependencies": "depcheck && yarn dedupe --check",
     "lint:dependencies:fix": "depcheck && yarn dedupe",
     "lint:eslint": "yarn build:only-clean && yarn ts-node ./scripts/run-eslint.ts --cache",
-    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix && yarn lint:dependencies:fix",
+    "lint:fix": "yarn lint:eslint --quiet --fix && yarn lint:misc --write && yarn constraints --fix && yarn lint:dependencies:fix",
```

Comment:
`--quiet` hides warnings, and the combination of `--quiet` and `--fix` will only autofix errors and not warnings.